_guides/upgrade.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: documentation
+title: How to upgrade to The Lounge v3
+---
+
+This guide lists all breaking changes made between the releases of v2.7.1 and v3.0.0 that may require you taking actions before upgrading and running The Lounge.
+
+In addition to the items listed in this guide, you may want to enable or configure new features of The Lounge v3 manually. For a more detailed list of all changes and additions, refer to the [release notes](https://github.com/thelounge/thelounge/releases/tag/v3.0.0) and [all merged pull requests](https://github.com/thelounge/thelounge/pulls?q=milestone%3A3.0.0+is%3Amerged+-label%3A%22Type%3A+Dependencies%22+-label%3A%22Meta%3A+Internal%22) for this release.
+
+{% include toc.md %}
+
+## TL;DR
+
+- You need Node.js v6.13.0 or more recent.
+- Everything called `[something]lounge[something]` becomes `[something]thelounge[something]`.
```

Comment:
I still think `[something]` is confusing to read here. Maybe just expand it to describe the command and folder separately?

Reply:
Agreed re: confusing. Will reword.